ansible/ansible-horizon-plugin-helper.yml

```diff
@@ -0,0 +1,74 @@
+---
+- block:
+
+  - name: Find horizon container name
+    find:
+      patterns: '*_horizon_*'
+      paths:
+        - /var/lib/lxc
+      file_type: directory
+    register: file_found
+
+  - debug: var=file_found
+
+  - set_fact: horizon_container_name="{{ (file_found.files[0]['path']).split('/')[4] }}"
```

Comment:
what if there are multiple horizon containers ?

Reply:
code added for multiple containers